apps/wfo-ui-surf/components/AppLogo/AppLogo.tsx

```diff
@@ -1,23 +1,27 @@
 import React, { ReactElement } from 'react';
 
-import Image from 'next/image';
+import { EuiText } from '@elastic/eui';
+import { useOrchestratorTheme } from '@orchestrator-ui/orchestrator-ui-components';
 
-import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { getAppLogoStyles } from '@/components/AppLogo/styles';
```

Comment:
Why are we getting the styles from the library but the logo from the app? I would say both from the same place so either from the library or the app. I would prefer to have the default logo be part of the library